databricks/koalas/tests/test_series.py

```diff
@@ -2364,13 +2364,16 @@ def test_factorize(self):
 
         pcodes, puniques = pser.factorize(sort=True, na_sentinel=-2)
         kcodes, kuniques = kser.factorize(na_sentinel=-2)
-
-        self.assert_eq(pcodes.tolist(), kcodes.to_list())
+        self.assert_eq(
+            [0, 1, 0, -2, -2] if is_lower_pandas_version else pcodes.tolist(), kcodes.to_list()
```

Comment:
This behavior difference only happens with pandas<0.24, right?
We should check with the actual pandas if the behavior is the same.

Reply:
Sounds good! Modified.